instrumentation/github.com/gin-gonic/gin/otelgin/option.go

```diff
@@ -88,3 +129,10 @@ func WithSpanNameFormatter(f func(r *http.Request) string) Option {
 		c.SpanNameFormatter = f
 	})
 }
+
+// WithRecordPanicConfig specifies a configuration for panic recording.
+func WithRecordPanicConfig(cfg RecordPanicConfig) Option {
```

Comment:
If we do want to keep this config, how about:

```suggestion
func WithRecordPanicConfig(enabled, stacktrace bool) Option {
```

Then, there's no need to have a config within a config.